server/lib/systemd/pbench-server.service.default

```diff
@@ -9,9 +9,13 @@ Documentation = https://github.com/distributed-system-analysis/pbench
 After=network.target httpd.service
 
 [Service]
+user = pbench
+group = pbench
 Type = simple
+PIDFile = /opt/pbench-server/gunicorn.pid
 ExecStart = /opt/pbench-server/bin/pbench-server
-ExecStop = /usr/bin/pkill pbench-server
+ExecStop = /bin/kill -s TERM $MAINPID
```

Comment:
What entity sets `MAINPID` to be the value stored in `/opt/pbench-server/gunicorn.pid`?

Reply:
`MAINPID` gets auto set via what we store inside `PIDFile` parameter of service file